optimizer_wrapper.rb

```diff
@@ -427,7 +466,7 @@ def self.split_independent_vrp(vrp)
     return [vrp] if (vrp.vehicles.size <= 1) || vrp.services.empty? # there might be zero services
 
     if vrp.services.all?{ |s| s.sticky_vehicles.size == 1 }
-      return split_independent_vrp_by_sticky_vehicle(vrp)
+      return split_independent_vrp_by_relation_and_sticky(vrp)
```

Comment:
I think we should do something similar with _split_independent_vrp_by_skills_ and convert it into something like _split_independent_vrp_by_relation_and_skills_. Because now  _split_independent_vrp_by_skills_ might also split vehicles from a common relation.